source/unified-test-format/tests/valid-pass/initialCollectionData-collectionOptions.yml

```diff
@@ -0,0 +1,39 @@
+description: initialCollectionData-collectionOptions.yml
```

Comment:
I think this should just be `initialCollectionData-collectionOptions`. Please double-check that none of the other files have the extension in their description.

Reply:
Done.